lib/puppetserver/ca/action/list.rb

```diff
@@ -176,8 +227,11 @@ def parse(args)
 
           errors_were_handled = Errors.handle_with_usage(@logger, errors, parser.help)
 
-          exit_code = errors_were_handled ? 1 : nil
-
+          if errors_were_handled
+            exit_code = 1
+          else
+            exit_code = nil
```

Comment:
I know this was here before, but didn't we talk about how maybe this should be `0` instead of `nil`? since `nil` isn't an exit code??

Reply:
I tried changing it to 0 and tested it.  The way that we have thing set up over `cli.rb` would require the `exit_code` here to not be a number.

Reply:
ahh, gotcha. we can leave it then.

Reply:
https://github.com/puppetlabs/puppetserver-ca-cli/blob/58c082b7fe8ac77bd8cf267795577e25b17bfc85/lib/puppetserver/ca/cli.rb#L94